tests/api/v1/userTokens/revokeRestore.js

```diff
@@ -112,6 +116,30 @@ describe('tests/api/v1/userTokens/revokeRestore.js, ' +
     });
   });
 
+  it('try to use a revoked token', (done) => {
+    api.post(`${path}/${uname}/tokens/${tname}/revoke`)
+    .set('Authorization', predefinedAdminUserToken)
+    .send({})
+    .expect(constants.httpStatus.OK)
+    .end((err, res) => {
+      if (err) {
+        return done(err);
+      }
+
+      expect(res.body).to.have.property('name', tname);
+      expect(res.body.isRevoked > '0').to.be.true;
+      api.get(path)
+      .set('Authorization', voldemortToken)
+      .send({})
+      .expect(constants.httpStatus.FORBIDDEN)
+      .end((err2, res2) => {
+        expect(res2.body.errors[0])
+          .to.have.property('description', 'Authentication Failed');
```

Comment:
Looks like you are not updating the description when throwing the error - but we have the description here. Is this error coming from somewhere else?

Reply:
Yeah I realized that verifyToken was swallowing that explanation and replacing it with generic "Authentication Failed" anyway, so I just got rid of the custom description there.